pywbem/_server.py

```diff
@@ -735,31 +735,34 @@ def get_central_instances(self, profile_path, central_class=None,
         * Central class methodology
         * Scoping class methodology
 
-        Use of the scoping class methodology requires specifying the central
-        class, scoping class and scoping path defined by the profile. If any
-        of them is `None`, this method will attempt only the
-        GetCentralInstances and central class methodologies, but not the
-        scoping class methodology.
-        If using these two methodologies does not result in any central
-        instances, and the scoping class methodology cannot be used,
-        :exc:`~pywbem.ModelError` is raised.
-
-        The scoping path is a directed traversal path from the central
-        instances to the scoping instances. Its first list item is always
-        the association class name of the traversal hop starting at the
-        central instances. For each further traversal hop, the list contains
-        two more items: The class name of the near end of that hop, and
-        the class name of the traversed association.
-        As a result, the class names of the central instances and scoping
-        instances are not part of the list.
-
-        Example for a 1-hop traversal:
+        All three methodologies start from the CIM_RegisteredProfile instance
+        referenced by the `profile_path` parameter.
+
+        Autonomous profiles implement the GetCentralInstances methodology
```

Comment:
Is this True,  I thought autonomous profiles must implement the Central instance methodology.  Or is this simply that the scoping methodology requires that autonomous profiles do the ECTP.  In any case the phrasing methodology vs. defining exactly what is implemented implies that the reader has knowledge of the PRP and we should probably not assume that for clients.

Reply:
Or possibly  just state that implement the scoping profile and are required to have an ECTP to central instances

Reply:
It is true that autonomous profiles must implement the central class methodology. The GetCentralInstances methodology is an optional addition. I.e. an autonomous profile does not have a choice to implement GetCentralInstances and then not to implement the central class methodology. I corrected that.

On your questions: The scoping profile of a component profile can be another component profile. In that case, it again has the usual choices.